media/css/firefox/releasenotes.scss

```
@@ -31,11 +31,16 @@ $image-path: '/media/protocol/img';
         }
     }
 
-    .download-button {
+    blockquote + .download-button {
         @media #{$mq-md} {
             float: right;
         }
     }
+
+    .mzp-c-call-out-compact {
+        background-color: #fff;
+        margin-top: 0px;
```

tests are upset because of the unit value on 0, quick fix:

```suggestion
        margin-top: 0;
```